### ISSUE TITLE
Dubious "Unused use" case

### ISSUE THREAD

**Issue by Fuco1** [slevomat/coding-standard#160]
Hi. 

Imagine a class like

```php
<?php

namespace App;

use Other;


class One
{

    /**
     * @var Other\Two
     * @inject
     */
    public $property;

}
```

The injected property might be used somewhere else (let's pretend `One` is a magic class somewhow), but the real issue is that `use Other` is tagged as unused.  I have the annotation parsing enabled for this sniff.

I think it assumes that because it is at the "root" and the annotation also starts with `Other` at the root so it is not necessary.  However, this is not always the case.  Imagine there would be a class `\App\Other\Two`.  Then the use clause actually distinguishes between that and `\Other\Two`.  I think the use here should only be declared useless if the annotation uses absolute namespace (starting with `\`)

I have created a little project you can use to check the various cases

    git clone https://github.com/Fuco1/slevomat-example
    cd slevomat-example
    composer update
    composer phpcs

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/f750828795da8dd9bb24b74bfce2700a94b8f660
